runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/IntegrationSections/IntegrationSections.module.scss

```diff
@@ -0,0 +1,160 @@
+@import "#src-landing/scss/main.scss";
+
+$ARROW_ICON_WIDTH: 300px;
+
+.root {
+  background-color: $palette-accent-primary;
```

Comment:
keep styles order, position should be higher

Reply:
done